docs(testing): update README adding karma webpack doc (#759)
open-wc/open-wc: packages/testing/README.md

});
```

> **__NOTE:__** if you are using `karma-webpack` you have to add `chai/chai.js` to your files in the `karma.conf.js`

```javascript
files: [
  { pattern: require.resolve('chai/chai.js' }, 
  // other files
]
```

> > and replacing in your test files where you `import { chai } from '@open-wc/testing` by referencing chai on window (if it applies).

Reviewer: ```suggestion
> If your test files use `import { chai } from '@open-wc/testing`, replace those imports with referencing chai on window.
```

Reviewer 2: question is do we still want to update/fix this? e.g. allow to import chai from `@open-wc/testing`?

Reviewer: I think we should, however, it's a bit awkward to export a global. if the user doesn't set up chai, do we throw an error?

Reviewer 3: We already import chai in our module, it's just that it doesn't work with webpack out of the box
